todo_app/.env.test

```diff
@@ -0,0 +1,9 @@
+# Flask server configuration.
+FLASK_APP=todo_app/app
+FLASK_ENV=Testing
+FLASK_SKIP_DOTENV=True
+
+# Change the following values for local development.
+SECRET_KEY=secret-key
+TRELLO_KEY=sadasdasdadqweqweqwe45345
```

Comment:
I hope these are not valid trello credentials :p

Reply:
no this is not a valid credential, should I leave it blank